dojo/jira_link/views.py

```diff
@@ -77,7 +77,7 @@ def webhook(request, secret=None):
                     raise Http404('No finding, finding_group or engagement found for JIRA issue {}'.format(jissue.jira_key))
 
                 assignee = parsed['issue']['fields'].get('assignee')
-                assignee_name = assignee['name'] if assignee else None
+                assignee_name = assignee.get('name')
```

Comment:
Per the issue, the `displayName` could be present instead. I think the following would accommodate both cases 

```suggestion
                # First look for the 'name' field. If not present, try 'displayName'. Else put None
                assignee_name = assignee.get('name', assignee.get('displayName', None))
```